Dockerfile

```diff
@@ -1,6 +1,11 @@
 # syntax=docker/dockerfile:1
-FROM mcr.microsoft.com/devcontainers/javascript-node:0-18 AS build
+FROM node:18.16.0-alpine AS build
 ARG REACT_APP_BACKEND_URL
+ARG REACT_APP_BASENAME
+ARG REACT_APP_DEBUG_MODE
+ARG REACT_APP_GITHUB_API_URL
+ARG REACT_APP_SEARCH_SOURCE
```

Comment:
Is it that these ARGs are truly "secrets"? If not, we could set them as ENV. Also, the GitHub Action that builds and pushes an image to our organization's registry doesn't allow dynamically setting of ARG (e.g --build-arg)

Reply:
Indeed they are not true secrets. They are more like configuration variables. I removed them from the Dockerfile. Although I'm not too sure how you would set them as ENV. Note that In the `.dockerignore` file I ignored the .env file.